changelog.d/20231127_164552_mzhiltsov_fix_skeleton_comparison.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+- Incorrect handling of the hidden points in skeletons in quality comparisons
```

Comment:
AFAICS, the first change in `quality_reports.py` isn't related to skeletons, so it probably deserves another entry.